docs/src/main/paradox/discovery.md

```diff
@@ -0,0 +1,128 @@
+<a id="discovery"></a>
+# Service Discovery
+
+Akka Discovery provides a simple interface around various ways of locating services, such as DNS
+or using configuration or key-value stores like zookeeper, consul etc.
+
+## What is Service Discovery
+
+Akka's Service Discovery talks specifically about discovering hosts and ports that relate to some
+logical name of a service.
+
+If you're looking for a way to discover Actors in a Cluster, you may want to look at the Receptionist
+pattern from Akka Typed instead. Since it provides a more fine-tuned towards Actors mechanism of
+doing the discovery.
+
+# Implementations
+
+TODO: Discussion about DNS vs other key-value stores.
+
+## Akka DNS Discovery
+
+The simplest, and also most natural form of service discovery is to use DNS as the source of truth regarding available 
+services. In the simplest version, we query for a service name -- which each cluster manager, such as Kubernetes, Mesos 
+or others define using their own naming schemes, and expect to get back a list of IPs that are related to this service.
```

Comment:
Two sentences in a row starting with "... simplest". Maybe the second doesn't really need it?